foreman/foreman.spec

```diff
@@ -196,6 +199,17 @@ Meta Package to install hammer rubygems and its dependencies
 
 %files cli
 
+%package debug
```

Comment:
There's a requires in 'foreman' for rsync, could you move it to the debug package please, as it's used by foreman-debug.